dor/adapters/technical_metadata.py

```diff
@@ -49,7 +49,7 @@ class TechnicalMetadata:
     valid: bool
 
     @classmethod
-    def create(cls, file_path: Path) -> Self:
+    def create(cls, file_path: Path) -> "TechnicalMetadata":
```

Comment:
Where you getting squiggles with `Self`? Or is this a pattern we should document?